boot/freeldr/freeldr/freeldr.c

```diff
@@ -98,3 +98,29 @@ int __cdecl iswctype(wint_t wc, wctype_t wctypeFlags)
 {
     return _isctype((char)wc, wctypeFlags);
 }
+
+#ifdef _MSC_VER
+#pragma warning(disable:4164)
+#pragma function(pow)
+#pragma function(log)
+#pragma function(log10)
+#endif
+
+// Stubs to avoid pulling in data from CRT
```

Comment:
What about building a CRT tailored for the kernel and/or freeldr, that doesn't use any of those functions?

Reply:
I guess we could do that, but that would mean 3 CRTs: CRT, LIBCNT_UM, LIBCNT_KM. Preferably this should avoid recompiling code, so you would have chains: CRT_COMMON<-CRT / CRT_COMMON<-LIBCNT_COMMON<-LIBCNT_[KM|UM]. That can quickly get messy. I might look at this in the future, but for this PR it's just too much extra hassle.